### ISSUE TITLE
[Bug] Using Safari on a MAC OPERATING system vm found that the chart does not work properly

### ISSUE THREAD

**Issue by damocles-s** [apache/echarts#17373]
### Version

5.3.2

### Link to Minimal Reproduction

https://codepen.io/cheung-shih/pen/abYmQjb

### Steps to Reproduce

In a VIRTUAL machine running MacOS operating system, Use safari browser to open the code.

### Current Behavior

1. I have a virtual machine running MacOS operating system, and I tried to simulate ajax requests on this machine to draw charts with different data per second.
2. when I use Safari on this machine, I find that the charts don't change and the data doesn't show up, but when I use Google Chrome everything works fine.
3. I tried not using a virtual machine, Google and Firefox both worked fine, but for some personal reason so i have no way to try using Safari outside of a virtual machine.

### Expected Behavior

I wanted Safari to be able to draw diagrams as well as Google chrome.

If can‘t, can tell me why?

### Environment

```markdown
- OS: macOS Monterey (12.0.1)
- Browser: Safari 15.1(17612.2.9.1.20)、Chrome(80.0.3987.149)
- Framework: Vue@2.6.10
```

### Any additional comments?

The VM configuration is as follows:
  memory: 14GB
  CPU: 8
  HDD(SATA): 100GB
  Network Adapter: Bridge mode (automatic)

**Comment by Ovilia** [comment]
There is no way to know what's happening based on the information provided. 

I don't have a virtual machine. I testes it on Safari on MacOS without VM and it works fine.

Here are what you can do:

Open the console on VM Safari and see if there is any error. Debug the related code and step in to see what's going on wrong.
If there is no error, you can change your test case to see what special setting and trigger the problem.
Basically, it's your own responsibility to do the job to see what's going wrong. People from the community can only give you some help.

**Comment by cheung-shih** [comment]
> 根据提供的信息，无法知道发生了什么。
> 
> 我没有虚拟机。我在没有 VM 的 MacOS 上的 Safari 上对其进行了测试，它运行良好。
> 
> 以下是您可以执行的操作：
> 
> 在 VM Safari 上打开控制台，看看是否有任何错误。调试相关代码并介入以查看发生了什么问题。 如果没有错误，您可以更改您的测试用例，看看有什么特殊设置并触发问题。 基本上，您有责任完成这项工作，看看出了什么问题。社区的人只能给你一些帮助。

谢谢您的回答。
虚拟机里面的safari浏览器和谷歌控制台没有任何报错。
在虚拟机外的safari是能够正常工作的。
目前还没有发现触发机制，怀疑单纯是在操作系统是macOs的safari中会出现图表显示异常的问题，是否和定时器异步获取数据有关？

**Comment by small-vagetable** [comment]
I have the same problem. Is there a solution?
